Pkgfile

```diff
@@ -4,7 +4,7 @@ format: v1alpha2
 
 vars:
   PKGS_PREFIX: ghcr.io/siderolabs
-  PKGS_VERSION: v1.3.0-alpha.0-39-g1e8df44
+  PKGS_VERSION: v1.3.0
   LINUX_FIRMWARE_VERSION: "20221109" # update this when updating PKGS_VERSION above
```

Comment:
didn't we update the linux firmware?

Reply:
nvm, already updated here

Reply:
not this time. I updated it last week

Reply:
https://github.com/siderolabs/extensions/commit/b35af1df6373cbecc99c9b330e0d27672bbc2d48#diff-4dedc27acfbaae8cce2bf60065f00438edce56ae56b22ed46d099f91f0f1fd15L8